src/containers/Vaults/VaultDetails.tsx

```diff
@@ -103,7 +103,7 @@ const VaultDetails = ({ ...props }) => {
         return () => {
             safeActions.setSingleSafe(null)
         }
-    }, [safe, safeActions, geb, liquidationData, safeActions.liquidationData])
+    }, [safe, safeActions, geb, liquidationData, safeActions.setLiquidationData])
```

Comment:
Please provide the reason for this change

Reply:
The liqduidationData field isn't a member of the safeActions object so it was throwing a type error. Not sure why it wasn't surfacing before.
I can change it to safeState.liquidationData if that was the intended behaviour. 

![image](https://github.com/open-dollar/od-app/assets/90171376/5384c81c-3b2a-4fd3-b48c-52a4d79e4132)